src/platform/silabs/rs911x/BLEManagerImpl.cpp

```diff
@@ -1101,6 +1123,37 @@ void BLEManagerImpl::StartBleAdvTimeoutTimer(uint32_t aTimeoutInMs)
     }
 }
 
+
+void BLEManagerImpl::BleSendIndicationTimeoutHandler(TimerHandle_t xTimer)
+{
+        ChipLogProgress(DeviceLayer, "BleSendIndicationTimeoutHandler::Start");
+        sInstance.HandleSoftTimerEvent();
+}
+
+void BLEManagerImpl::CancelBleSendIndicationTimeoutTimer(void)
+{
+    if (xTimerStop(sbleSendIndicationTimeoutTimer, pdMS_TO_TICKS(0)) == pdFAIL)
+    {
+        ChipLogError(DeviceLayer, "Failed to stop BledAdv timeout timer");
+    }
+}
+
+void BLEManagerImpl::StartBleSendIndicationTimeoutTimer(uint32_t aTimeoutInMs)
+{
+    if (xTimerIsTimerActive(sbleSendIndicationTimeoutTimer))
+    {
+        CancelBleAdvTimeoutTimer();
+    }
+
+    // timer is not active, change its period to required value (== restart).
+    // FreeRTOS- Block for a maximum of 100 ticks if the change period command
+    // cannot immediately be sent to the timer command queue.
+    if (xTimerChangePeriod(sbleSendIndicationTimeoutTimer, pdMS_TO_TICKS(aTimeoutInMs), pdMS_TO_TICKS(100)) != pdPASS)
```

Comment:
`pdMS_TO_TICKS(100)` this 100ms is not a predefined variable?

Reply:
Updated